tools/importer-rest-api-specs/components/dataapigeneratorjson/models/operations.go

```diff
@@ -0,0 +1,73 @@
+package models
+
+type Operation struct {
+	// Name specifies the Display Name for this Operation (e.g. `Create`) which is also
+	// valid as an identifier.
+	Name string `json:"Name"`
+
+	// ContentType specifies the format of the information being sent with the Operation (e.g. `application/json; charset=utf-8`)
+	ContentType string `json:"ContentType,omitempty"`
+
+	// ExpectedStatusCodes specifies is a list of Status Codes which are expected to be returned (e.g. 200, 201)
+	ExpectedStatusCodes []int `json:"ExpectedStatusCodes,omitempty"`
+
+	// FieldContainingPaginationDetails is a reference to the field within the Response
+	// which contains the pagination details, (e.g. `nextLink`)
+	FieldContainingPaginationDetails string `json:"FieldContainingPaginationDetails,omitempty"`
```

Comment:
This is optional/should be a pointer

```suggestion
	FieldContainingPaginationDetails *string `json:"FieldContainingPaginationDetails,omitempty"`
```